opm/test_util/EclFilesComparator.cpp

```diff
@@ -108,15 +108,15 @@ void ECLFilesComparator::deviationsForCell(double val1, double val2, const std::
         std::cout << "(value1, value2) = (" << val1 << ", " << val2 << ")\n\n";
     }
     if (val1 < 0) {
-        if (abs(val1) > absTolerance) {
+        if (std::abs(val1) > absTolerance) {
```

Comment:
it's a real issue. the problem is that abs(x) resolves to the function with signature 
```
int abs(int x)
```
i.e. it applies a silent implicit cast to x. this depends on which headers happen to be included. it bit here on my toolchain.